gatsby-node.js

```diff
@@ -3,9 +3,10 @@ const path = require(`path`)
 const fetch = require("node-fetch")
 
 const PROD_HOST = "open-disclosure-api.codeforsanjose.com"
-const HOSTNAME =
-  process.env.GATSBY_API_HOST ||
-  (process.env.production ? PROD_HOST : "localhost:5000")
+// const HOSTNAME =
```

Comment:
If we want to just always use the deployed API then we can just delete this commented out code